### ISSUE TITLE
Training loss curve on MMC4 dataset?

### ISSUE THREAD

**Issue by tonylins** [mlfoundations/open_flamingo#226]
Hi, thanks for the great work! I tried training on a subset of MMC4-core but the LM loss does not go down too much. Is it possible to share the MMC4 loss curve for reference, so that I may know if it is expected (or potentially a bug). Thanks so much!

**Comment by FingerRec** [comment]
Meet the same problem, it seems the training loss on MMC4 is hard to convergence.

**Comment by anas-awadalla** [comment]
<img width="468" alt="Screenshot 2023-07-23 at 9 09 57 PM" src="https://github.com/mlfoundations/open_flamingo/assets/55931244/5e950b15-1e4a-41f3-b777-9f1dd332f537">

Here are the loss plots for some of our training runs. We also find that the loss on MMC4 decreases more slowly than the loss on LAION. We anticipate that this could be the case because we use a pre-trained language model, which is already a strong predictor of the next token.

**Comment by FingerRec** [comment]
Thanks. What's the sim thresh score used for this figure?

**Comment by i-gao** [comment]
These curves use a threshold of 0.24.

**Comment by FingerRec** [comment]
Thanks. In addition, have you ever show the validation loss before?  If I use a subset of mmc4 for pretrain (around 1m website). The validation loss begin to rise up in very short iterations. 

Train:
![image](https://github.com/mlfoundations/open_flamingo/assets/20349006/c4f0767c-a381-49bb-967d-55e3e71bbb63)

Val:
![image](https://github.com/mlfoundations/open_flamingo/assets/20349006/e518425a-3d6c-471c-8f71-1a74a44eea52)

**Comment by i-gao** [comment]
Hmm, we haven't plotted such a validation loss before -- this behavior is pretty surprising to me! Do you know if your downstream performance on task benchmarks improves or degrades with training?

**Comment by FingerRec** [comment]
The downstream performance are also unstable. The ckpt in middle sometimes better than the final. I guess it's because I train the model with only 1M LAION and 1M MMc4. The data scale is too small. How many LAION and MMC4 samples you used for the above figure?

**Comment by i-gao** [comment]
Ah, okay! The x-axis in the training curve plots refer to the number of interleaved (mmc4) samples.
